### ISSUE TITLE
Chat Title

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#159]
```
Feature Request of adding a chat window title when using the "chat:contact" 
cmd.  If I am not mistaken, the separate chat window's title is the address 
used to create the tab.  Instead of having many chat tabs that all look alike, 
this will allow users to determine which tab is for which contact.

Right now the window's title is:
gtalksms_#########_myemail@conference...

Is there a way to include the contact's name in this area? 
ContactName_gtalksms_#######_myemail@congerence...
```

Original issue reported on code.google.com by `nat...@mynarcik.com` on 4 May 2011 at 3:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 11 May 2011 at 6:31
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Note: We already set the MUC topic to the name and/or number of the relevant 
contact. But I see the ineffectiveness here because most XMPP client will only 
show the JID of the MUC in the title bar of the MUC Chat.
The JID in this case is currently a fixed string with a random number. This was 
chosen for privacy reasons, as some XMPP Servers will list their MUCs publicly. 
Including the name and/or number would reveal the name and even worser also the 
phone number your contact on servers which list their MUC rooms publicly and 
don't obey the "isPrivateRoom" XMPP flag, which we also try to set.
```

Original comment by `fschm...@gmail.com` on 21 May 2011 at 5:06
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 10 Jun 2012 at 11:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
